openpilot dmonitoring model does pass compile self-tests in latest tinygrad with QCOM=1

```
wget https://github.com/commaai/openpilot/raw/v0.9.7/selfdrive/modeld/models/dmonitoring_model.onnx -O /tmp/monitor.onnx
PYTHONPATH=":/data/openpilot/tinygrad_repo" QCOM=1 python3 /data/openpilot/tinygrad_repo/examples/openpilot/compile3.py /tmp/monitor.onnx
```

produces
```
comma@comma-8fcff1ab:/data/openpilot$ PYTHONPATH=":/data/openpilot/tinygrad_repo" QCOM=1 python3 /data/openpilot/tinygrad_repo/examples/openpilot/compile3.py /tmp/monitor.onnx
loaded model
created tensors
run 0
opened device CLANG from pid:74679
scheduled 586 kernels
memory reduced from 40.11 MB -> 33.66 MB, 321 -> 290 bufs
run 1
scheduled 322 kernels
JIT captured 334 kernels with 2 inputs
pruned from 334 -> 141 kernels
JIT memory reduced from 30.29 MB -> 23.83 MB, 132 -> 101 bufs
run 2
JIT GRAPHing batch with 133 kernels on device <tinygrad.runtime.ops_qcom.QCOMDevice object at 0x7f9036bcd0>
*** CLANG      1 copy        4,   CLANG <- QCOM            arg  2 mem  0.05 GB tm    126.25us/     0.13ms (     0.00 GFLOPS    0.0|0.0     GB/s) 
*** CLANG      2 copy        4,   CLANG <- QCOM            arg  2 mem  0.05 GB tm     83.64us/     0.21ms (     0.00 GFLOPS    0.0|0.0     GB/s) 
*** CLANG      3 copy        4,   CLANG <- QCOM            arg  2 mem  0.05 GB tm     83.07us/     0.29ms (     0.00 GFLOPS    0.0|0.0     GB/s) 
*** CLANG      4 copy        4,   CLANG <- QCOM            arg  2 mem  0.05 GB tm     78.44us/     0.37ms (     0.00 GFLOPS    0.0|0.0     GB/s) 
*** CLANG      5 copy        4,   CLANG <- QCOM            arg  2 mem  0.05 GB tm     78.54us/     0.45ms (     0.00 GFLOPS    0.0|0.0     GB/s) 
*** CLANG      6 copy        4,   CLANG <- QCOM            arg  2 mem  0.05 GB tm     95.83us/     0.55ms (     0.00 GFLOPS    0.0|0.0     GB/s) 
*** CLANG      7 copy        4,   CLANG <- QCOM            arg  2 mem  0.05 GB tm     89.63us/     0.64ms (     0.00 GFLOPS    0.0|0.0     GB/s) 
*** CLANG      8 copy        4,   CLANG <- QCOM            arg  2 mem  0.05 GB tm     91.66us/     0.73ms (     0.00 GFLOPS    0.0|0.0     GB/s) 
*** QCOM       9 <batched 133>                             arg  2 mem  0.05 GB tm     13.96ms/    14.68ms (   107.90 GFLOPS    5.4|138.0   GB/s) 
*** CLANG     10 copy      336,   CLANG <- QCOM            arg  2 mem  0.05 GB tm    102.86us/    14.79ms (     0.00 GFLOPS    0.0|0.0     GB/s) 
captured 141 kernels
Traceback (most recent call last):
  File "/data/openpilot/tinygrad_repo/examples/openpilot/compile3.py", line 78, in <module>
    test_val = compile() if not getenv("RUN") else None
               ^^^^^^^^^
  File "/data/openpilot/tinygrad_repo/examples/openpilot/compile3.py", line 48, in compile
    np.testing.assert_equal(test_val, ret)
  File "/usr/local/pyenv/versions/3.11.4/lib/python3.11/site-packages/numpy/testing/_private/utils.py", line 282, in assert_equal
    return assert_array_equal(actual, desired, err_msg, verbose)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/pyenv/versions/3.11.4/lib/python3.11/site-packages/numpy/testing/_private/utils.py", line 920, in assert_array_equal
    assert_array_compare(operator.__eq__, x, y, err_msg=err_msg,
  File "/usr/local/pyenv/versions/3.11.4/lib/python3.11/contextlib.py", line 81, in inner
    return func(*args, **kwds)
           ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/pyenv/versions/3.11.4/lib/python3.11/site-packages/numpy/testing/_private/utils.py", line 797, in assert_array_compare
    raise AssertionError(msg)
AssertionError: 
Arrays are not equal

Mismatched elements: 41 / 84 (48.8%)
Max absolute difference: 13.
Max relative difference: 198.11111
 x: array([[ 8.185687e-01,  1.055439e+00,  2.851110e-01,  5.819049e-01,
         5.845404e-02,  4.363403e-01, -2.348640e+00, -2.404555e+00,
        -2.629772e+00, -5.756116e+00, -5.598006e+00, -5.552535e+00,...
 y: array([[ 0.818569,  1.055439,  0.285111,  0.581905,  0.058454,  0.43634 ,
        -2.34864 , -2.404555, -2.629772, -5.756116, -5.598006, -5.552535,
         3.821026,  6.      ,  0.113281, -0.75    , -6.036133,  0.398438,...
```


Seems broken since this commit: 9eb6eef441b4688b5f07b6ac8738fe03de27e521

--- a/.github/workflows/benchmark.yml
+++ b/.github/workflows/benchmark.yml
@@ -500,6 +500,8 @@ jobs:
       run: PYTHONPATH="." QCOM=1 python3 examples/openpilot/compile3.py https://github.com/commaai/openpilot/raw/v0.9.7/selfdrive/modeld/models/supercombo.onnx
     - name: openpilot compile3 0.9.7+ tomb raider
       run: PYTHONPATH="." QCOM=1 python3 examples/openpilot/compile3.py https://github.com/commaai/openpilot/raw/e8bea2c78ffa92685ece511e9b554122aaf1a79d/selfdrive/modeld/models/supercombo.onnx
+    - name: openpilot dmonitoring compile3 0.9.7
+      run: PYTHONPATH="." QCOM=1 python3 examples/openpilot/compile3.py https://github.com/commaai/openpilot/raw/v0.9.7/selfdrive/modeld/models/dmonitoring_model.onnx
     - name: Run process replay tests
       run: cp test/external/process_replay/process_replay.py ./process_replay.py && git fetch origin master && git -c advice.detachedHead=false checkout origin/master && PYTHONPATH=. python3 process_replay.py
     - uses: actions/upload-artifact@v4

--- a/tinygrad/runtime/ops_qcom.py
+++ b/tinygrad/runtime/ops_qcom.py
@@ -219,7 +219,8 @@ class QCOMProgram(HCQProgram):
     self.max_threads = min(1024, ((384 * 32) // (max(1, (self.fregs + round_up(self.hregs, 2) // 2)) * 128)) * 128)
     device._ensure_stack_size(self.hw_stack_offset * 4)
 
-    super().__init__(QCOMArgsState, self.device, self.name, kernargs_alloc_size=2048 + (self.tex_cnt + self.ibo_cnt) * 0x40 + self.samp_cnt * 0x10)
+    kernargs_alloc_size = round_up(2048 + (self.tex_cnt + self.ibo_cnt) * 0x40 + self.samp_cnt * 0x10, 0x100)
+    super().__init__(QCOMArgsState, self.device, self.name, kernargs_alloc_size=kernargs_alloc_size)
 
   def __call__(self, *bufs, global_size:Tuple[int,int,int]=(1,1,1), local_size:Tuple[int,int,int]=(1,1,1), vals:Tuple[int, ...]=(), wait=False):
     if self.max_threads < prod(local_size): raise RuntimeError("Too many resources requested for launch")

